test/remote/gateways/remote_ixopay_test.rb

```diff
@@ -5,36 +5,43 @@ def setup
     @gateway = IxopayGateway.new(fixtures(:ixopay))
 
     @amount = 100
-    @credit_card = credit_card('4000100011112224')
+    @credit_card = credit_card('4111111111111111')
     @declined_card = credit_card('4000300011112220')
+
     @options = {
       billing_address: address,
-      description: 'Store Purchase'
+      shipping_address: address,
+      email: 'test@example.com',
+      description: 'Store Purchase',
+      ip: '192.168.1.1',
     }
   end
 
   def test_successful_purchase
     response = @gateway.purchase(@amount, @credit_card, @options)
+
     assert_success response
-    assert_equal 'REPLACE WITH SUCCESS MESSAGE', response.message
-  end
+    assert_equal 'FINISHED', response.message
+    assert_match(/[0-9a-zA-Z]+(|[0-9a-zA-Z]+)*/, response.authorization)
```

Comment:
What is this intended to check? Would it be just as effective to say something like `assert_not response.authorization.blank?` ?